src/theme/components/PaperTheme.ts

```diff
@@ -1,11 +1,14 @@
-import { Components } from "@mui/material"
+import { Components, Theme } from "@mui/material"
 
-export default function PaperTheme(): Components {
+export default function PaperTheme(theme: Theme): Components {
   return {
     MuiPaper: {
       styleOverrides: {
         root: {
-          border: "1px solid #000000",
+          border:
+            theme.palette.mode === "light"
+              ? theme.palette.other?.divider
```

Comment:
should be `1px solid ${theme.palette.other?.divider}`

Reply:
Exactly 😊